wagtail/images/models.py

```diff
@@ -432,6 +440,8 @@ def run(self, image, output):
                 return willow.save_as_png(output, optimize=True)
             elif output_format == 'gif':
                 return willow.save_as_gif(output)
+            elif output_format == 'webp' and hasattr(willow, 'save_as_webp'):
+                return willow.save_as_webp(output)
```

Comment:
webp support will come in the next Willow version. before that this could change as:
```python
elif output_format == 'webp':
    if hasattr(willow, 'save_as_webp'):
       return willow.save_as_webp(output)
    # Raise warning when webp was supported but we are falling back to png
    return willow.save_as_png(output)
```